JapaneseFilter/sections/css_extended.txt

```diff
@@ -552,7 +552,8 @@ katuru.com#?#.ui-block-b:has(script:not([src]))
 warotanikki.com#?#.widget_diver_widget_pcsp > div > div.adp:upward(2)
 warotanikki.com#?#.widget_diver_widget_pcsp > div > center > script[src$="/adstir.js"]:upward(3)
 blogterest.net#?#.blockSide > div.blockWrapper > div > div[class$="eroterest_partsv2"]:upward(2)
-39.benesse.ne.jp#?#.newPostWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)
+39mag.benesse.ne.jp,39.benesse.ne.jp#?#.newPostWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)
+39mag.benesse.ne.jp#?#.postListWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)
```

Comment:
Unite them to single rule `#?#li[class$="_item"] > div[id^="div-M_MD_gtx_"]:upward(1)`

Reply:
OK!